app/helpers/turbo/streams_helper.rb

```diff
@@ -40,8 +40,8 @@ def turbo_stream
   # (when Current.account.id = 5). Updates to this stream can be sent like
   # <tt>entry.broadcast_append_to entry.account, :entries, target: "entries"</tt>.
   def turbo_stream_from(*streamables, **attributes)
-    attributes["channel"] = "Turbo::StreamsChannel"
-    attributes["signed-stream-name"] = Turbo::StreamsChannel.signed_stream_name(streamables)
+    attributes[:channel] = "Turbo::StreamsChannel"
+    attributes[:"signed-stream-name"] = Turbo::StreamsChannel.signed_stream_name(streamables)
```

Comment:
I was seeing "TypeError: hash key "channel" is not a Symbol" failing/error tests on:

BroadcastsTest#test_Users::Profile_broadcasts_Turbo_Streams
BroadcastsTest#test_Message_broadcasts_Turbo_Streams
Turbo::StreamsHelperTest#test_with_streamable
Turbo::StreamsHelperTest#test_with_streamable_and_html_attributes

https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag suggests that these should be symbols, so after this change the tests were passing for me. I'm not sure why this wasn't failing before. If I missed something let me know and I can revert this change.